src/main/java/fr/openent/viescolaire/service/impl/DefaultGroupingService.java

```diff
@@ -159,4 +159,22 @@ public Future<JsonObject> deleteGrouping(String groupingId) {
         }));
         return promise.future();
     }
+
+    @Override
+    public Future<JsonObject> deleteGroupingAudience(String groupingId, String studentDivisionId) {
+        Promise<JsonObject> promise = Promise.promise();
+        JsonArray values = new JsonArray();
+        String query = "DELETE FROM " + TABLE_REL + " WHERE " + TABLE_REL + ".grouping_id = ? AND " + TABLE_REL + ".student_division_id = ?";
```

Comment:
Simplifiable par : "DELETE FROM " + TABLE_REL + " WHERE grouping_id = ? AND student_division_id = ?";

Reply:
oui mais en soit explicite